spec/features/index_spec.rb

```diff
@@ -4,20 +4,19 @@
 
 RSpec.describe "Index page", type: :feature do
 
-
     it "includes the Smartflix header" do
         visit "/"
         expect(page).to have_text("Smartflix")
     end
 
    context "when titles are given" do
-
     it "includes the shows titles" do 
       visit "/" 
+      
```

Comment:
instead of doing this what you could do here is make a more explicit spec. For example:
```
   visit "/"
   expect(page).to have_text("Dick Johnson Is Dead")(which is the first title)
   expect(page).to_not have_text("Dharmakshetra")(which is the 51st title)
```
Because this way you have slowing down specs a lot by having to parse CSV which is not necessary and also you are not specing the ".first(50)" logic by doing it this way.

Reply:
Yes, that makes sense 👍 Thanks Goran